addons/backpacks/functions/fnc_backpackToBack.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Author: marc_book
+ * Put's backpack to normal position.
+ *
+ * Arguments:
+ * 0: Player <OBJECT>
+ *
+ */
+
+#include "script_component.hpp"
+
+_unit = _this select 0;
+
+hint format ["%1", attachedObjects _unit];
+
+_backpack = (attachedObjects _unit) select 0;
```

Comment:
If user has lights attached, the backpack might not be the first in attachedObjects